### PR TITLE
remove trailing whitespace

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

**Comment by brammool** [comment]
I'll include it.  Except the change in change.txt, the trailing spaces there are intentional.
Also the first one in map.txt and a few others.

File: runtime/doc/change.txt [hunk 1]
```diff
@@ -1235,7 +1235,7 @@ There are ten types of registers:		*registers* *{register}* *E354*
 5. Three read-only registers ":, "., "%
 6. Alternate buffer register "#
 7. The expression register "=
-8. The selection and drop registers "*, "+ and "~ 
+8. The selection and drop registers "*, "+ and "~
```
Comment (k-takata) [inline runtime/doc/change.txt:1238]: This is intentional. Because a line that ends with `~` will be highlighted.

File: runtime/doc/change.txt [hunk 2]
```diff
@@ -1340,7 +1340,7 @@ If the "= register is used for the "p" command, the String is split up at <NL>
 characters.  If the String ends in a <NL>, it is regarded as a linewise
 register.
 
-8. Selection and drop registers "*, "+ and "~ 
+8. Selection and drop registers "*, "+ and "~
```
Comment (k-takata) [inline runtime/doc/change.txt:1343]: Same as above.

File: runtime/doc/change.txt [hunk 3]
```diff
@@ -1842,9 +1842,9 @@ editing text paragraphs.  A few hints on how to use this:
 
 - Set 'formatoptions' to "aw2tq" to make text with indents like this:
 
-	    bla bla foobar bla 
+	    bla bla foobar bla
 	bla foobar bla foobar bla
-	    bla bla foobar bla 
+	    bla bla foobar bla
 	bla foobar bla bla foobar
```
Comment (k-takata) [inline runtime/doc/change.txt:1848]: This is intentional. See the above paragraphs of this help.

File: runtime/doc/intro.txt [hunk 4]
```diff
@@ -899,10 +899,10 @@ buffer lines	logical lines	window lines	screen lines ~
 						8.  ccc ccc c
 1. aaa		1. aaa		1. aaa		9.  cc
 2. bbb		2. bbb		2. bbb		10. ddd
-3. ccc ccc ccc	3. ccc ccc ccc	3. ccc ccc c	11. ~ 
+3. ccc ccc ccc	3. ccc ccc ccc	3. ccc ccc c	11. ~
 4. ddd		4. ddd		4. cc		12. === status line ===
 				5. ddd		13. (command line)
-				6. ~ 
+				6. ~
```
Comment (k-takata) [inline runtime/doc/intro.txt:905]: intentional

File: runtime/doc/map.txt [hunk 5]
```diff
@@ -175,7 +175,7 @@ This unmap command does NOT work: >
 
 Because it tries to unmap "@@ ", including the white space before the command
 separator "|".  Other examples with trailing white space: >
-	unmap @@ 
+	unmap @@
```
Comment (k-takata) [inline runtime/doc/map.txt:178]: This is intentional as explained in this paragraph.

File: runtime/doc/options.txt [hunk 6]
```diff
@@ -5103,7 +5103,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	The cursor is displayed at the start of the space a Tab character
 	occupies, not at the end as usual in Normal mode.  To get this cursor
 	position while displaying Tabs with spaces, use: >
-		:set list lcs=tab:\ \ 
+		:set list lcs=tab:\ \
```
Comment (k-takata) [inline runtime/doc/options.txt:5106]: This is intentional as explained in this paragraph.

File: runtime/doc/options.txt [hunk 7]
```diff
@@ -7138,7 +7138,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 			feature}
 	String to put at the start of lines that have been wrapped.  Useful
 	values are "> " or "+++ ": >
-		:set showbreak=>\ 
+		:set showbreak=>\
```
Comment (k-takata) [inline runtime/doc/options.txt:7141]: This is intentional as explained in this paragraph.

File: runtime/doc/usr_12.txt [hunk 8]
```diff
@@ -298,7 +298,7 @@ Another wasteful use of spaces is placing them before a tab.  Often these can
 be deleted without changing the amount of white space.  But not always!
 Therefore, you can best do this manually.  Use this search command: >
 
-	/ 	
+	/
```
Comment (k-takata) [inline runtime/doc/usr_12.txt:301]: This is intentional as explained in this paragraph.

File: runtime/doc/usr_41.txt [hunk 9]
```diff
@@ -1861,7 +1861,7 @@ mapping.  For these commands, any character until the end-of-line or '|' is
 included.  As a consequence of this behavior, you don't always see that
 trailing whitespace is included: >
 
-	map <F4> o#include  
+	map <F4> o#include
```
Comment (k-takata) [inline runtime/doc/usr_41.txt:1864]: This is intentional as explained in this paragraph.

**Review by k-takata — COMMENTED** [review]
As already commented by @brammool, some of them are intentional.
You cannot remove trailing whitespace without checking whether it is intentional.

**Pull request closed without merging**